velero-plugins/pod/restore.go

```diff
@@ -113,9 +113,55 @@ func (p *RestorePlugin) Execute(input *velero.RestoreItemActionExecuteInput) (*v
 	destStagePodImage := input.Restore.Annotations[common.StagePodImageAnnotation]
 	if len(pod.Labels[common.IncludedInStageBackupLabel]) > 0 && len(destStagePodImage) > 0 {
 		p.Log.Infof("[pod-restore] swapping stage pod images for pod %s", pod.Name)
+		pvcVolumes := []corev1API.Volume{}
+		exclude := []string{}
+		moveVolumes := []string{}
```

Comment:
nit: not a merge blocker, but a minor comment. It's slightly confusing to call this "moveVolumes" since it also includes snapshot copy volumes. "exclude" is already in use, though. Maybe "excludePVC" or "excludeAnnotation" or something like that?

Reply:
+1, removed `excude` as it was not being used